compiler/dialects/snax_stream.py

```diff
@@ -21,6 +22,11 @@
 )
 from xdsl.parser import AttrParser
 from xdsl.printer import Printer
+from xdsl.traits import SymbolTable
+
+from compiler.accelerators.streamers.streamers import StreamerConfiguration
```

Comment:
Are you planning to put a lot more stuff into streamers? Or can we do 
```suggestion
from compiler.accelerators.streamers import StreamerConfiguration
```

Reply:
we can import like this! auto-import fucked this one up for me. I put streamers in a folder for possible variations on the streamer (sparse-streamer, pointer-chasing-streamer, ...)